.goreleaser.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  - GO111MODULE=on
+  - GOPROXY=https://gocenter.io
+before:
+  hooks:
+    - go mod download
+brews:
+  - github:
+      owner: fubarhouse
+      name: homebrew-pygmy-go
+    folder: Formula
+    homepage: "https://github.com/fubarhouse/pygmy-go"
+    description: "Amazee.io's local development helper tool"
+    skip_upload: false
+    test: system "#{bin}/pygmy-go version"
+    install: bin.install "pygmy-go"
+    dependencies:
+      - name: go
```

Comment:
Does this download a built binary and extract it - if so, is `go` a dependency, or does it build it locally?

Reply:
You can see an example of the results here:
https://github.com/fubarhouse/homebrew-pygmy-go/blob/master/dist/pygmy-go.rb

Reply:
yup - are those downloads usable binaries - or do they need building once tapped?

Reply:
They contain the compiled binaries, but I will be verifying this.
I also need to make sure those binaries contain the correct version number - but right now I am comfortable they do.

Reply:
we can always remove go locally to test if we need to - I think having this set will also download go via brew anyway.

Reply:
Agreed, the dependency should not be needed.
I'ill do some more testing tonight when I can actually work on this again.